### ISSUE TITLE
Crash when using C# 9 records

### ISSUE THREAD

**Issue by 304NotModified** [RandomEngy/UnitTestBoilerplateGenerator#6]
### Installed product versions
- Visual Studio: 2019
- This extension: 2.5.5

### Description
When generating with a `record`, 

### Steps to recreate
1. Create a record:
    ```c#
    using System;

    public record TestRecord(String TestProp);
    ```
2. Right click file: create Unit Test boilerplate
3. Press: Create unit test class
### Current behavior
We get the error "could not find class or struct"

![image](https://user-images.githubusercontent.com/5808377/127854194-9ac4fda1-8df9-4920-b1a2-e5ada5d7c679.png)

### Expected behavior
Generated testclass

**Comment by 304NotModified** [comment]
Guess this is 

https://github.com/RandomEngy/UnitTestBoilerplateGenerator/blob/1214531aab3ccde4349f283b5d24722d5355581f/src/Services/TestGenerationService.cs#L165

**Comment by RandomEngy** [comment]
Thanks. Looks like it needs some work to keep up with the new C# features. Been a bit short on time recently so I don't know when I might get to this. But always accepting PRs.

**Comment by 304NotModified** [comment]
I could give a try, but the unit tests arent running on AppVeyor?

**Comment by RandomEngy** [comment]
Testing is done through a custom debug mode tool. After you launch in debug mode there's a "UTBG Self-test" option in the Extensions menu. That runs the extension on code in the Sandbox solution and compares against expected values.

**Comment by 304NotModified** [comment]
Unfortunately that gives me:


```
System.Windows.Markup.XamlParseException
  HResult=0x80131501
  Message=The type initializer for 'UnitTestBoilerplate.ViewModel.SideBySideDiffModelVisualizer' threw an exception.
  Source=PresentationFramework
  StackTrace:
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadComponent(Object component, Uri resourceLocator)
   at UnitTestBoilerplate.View.SelfTestDialog.InitializeComponent() in D:\github\304NotModified\UnitTestBoilerplateGenerator\src\View\SelfTestDialog.xaml:line 1

Inner Exception 1:
FileNotFoundException: Could not load file or assembly 'DiffPlex, Version=1.4.1.0, Culture=neutral, PublicKeyToken=1d35e91d1bd7bc0f' or one of its dependencies. The system cannot find the file specified.

```

**Comment by RandomEngy** [comment]
That's weird. Do you see DiffPlex in the NuGet packages list?

![image](https://user-images.githubusercontent.com/9450861/127958064-9b236d96-8d2e-4653-9266-373deff194f3.png)

Did you run a nuget package restore? Do you see DiffPlex.dll in /src/bin/Debug after you build?

**Comment by 304NotModified** [comment]
Yes I did a restore (also retried) and the package/bin is there:

![image](https://user-images.githubusercontent.com/5808377/128600558-9e4960d8-40b2-413f-b243-d034ae6bce6e.png)

**Comment by 304NotModified** [comment]
For some reasons it all won't work the first time, but it does after some tries. 🤣

**Comment by 304NotModified** [comment]
I get locally spaces instead of tabs. Is that because of one of my VS2019 settings? See https://github.com/RandomEngy/UnitTestBoilerplateGenerator/pull/10

**Comment by RandomEngy** [comment]
I think you need to temporarily change your VS settings to use Tabs in order to run the self-test. It seems it's no longer picking up the project settings when doing the auto-format.

**Comment by RandomEngy** [comment]
C# 9 record support is available in 2.5.7.
